### PR TITLE
+xiph.org/flac

### LINKED ISSUE #232: +flac (100/548)
Formulae: https://formulae.brew.sh/formula/flac

### PR DESCRIPTION
**Pull request by getneil**
resolves #232

### CODE REVIEW

File: projects/xiph.org/flac/package.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+distributable:
+  url: https://downloads.xiph.org/releases/flac/flac-{{version}}.tar.xz
+  strip-components: 1
+
+versions:
+  github: xiph/flac/tags
```
Comment (mxcl) [inline projects/xiph.org/flac/package.yml]: use `xiph/flag/releases/tags` or we don't get updates

File: projects/xiph.org/flac/package.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  dependencies:
+    freedesktop.org/pkg-config: ^0.29
+    gnu.org/libtool: ^2.4
+    gnu.org/automake: ^1.16
+    gnu.org/autoconf: ^2.71
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+  script: |
+    ./configure $ARGS
+    make install
```
Comment (mxcl) [inline projects/xiph.org/flac/package.yml]: `make --jobs {{ hw.concurrency }} install` makes the build faster.

File: projects/xiph.org/flac/test.flac
Comment (mxcl) [inline projects/xiph.org/flac/test.flac:1]: fixture is too big. pantries are downloaded by every user including fixtures.

Use this one: 
[test.flac.zip](https://github.com/teaxyz/pantry/files/11011021/test.flac.zip)

**Review by mxcl — CHANGES_REQUESTED** [review]
thanks but a few changes

File: projects/xiph.org/flac/package.yml (hunk 1 above)
Comment (getneil) [reply projects/xiph.org/flac/package.yml]: applied.

File: projects/xiph.org/flac/test.flac
Comment (getneil) [reply projects/xiph.org/flac/test.flac:1]: applied.

**Pull request merged**